src/views/modal_handler.rs

```diff
@@ -198,16 +223,38 @@ impl DisplayModal {
                 .padding(15)
                 .into()
             }
-            Some(ModalType::Info { title, body }) => {
+            Some(ModalType::Info {
+                title,
+                body,
+                is_version,
+            }) => {
                 let text_style = TextStyle {
                     text_color: Color::new(0.447, 0.624, 0.812, 1.0),
                 };
-
-                let button_row = Row::new().push(
-                    button("Close")
-                        .on_press(Message::ModalHandle(ModalMessage::HideModal))
-                        .style(MODAL_CANCEL_BUTTON_STYLE.get_button_style()),
-                );
+                let mut hyperlink_row = Row::new().width(Length::Fill);
+                let mut button_row = Row::new();
+                if *is_version {
+                    hyperlink_row = hyperlink_row.push(Text::new("Full source available at: "));
+                    hyperlink_row = hyperlink_row
+                        .push(
+                            button(Text::new("Project Repository"))
```

Comment:
I'd put the link here, so the user knows where you are about to send them.
Either "github" or the full url "https://github...."?